### DISCUSSION TITLE
issue: Controlled inputs don't work with reset and resetField

### DISCUSSION THREAD

**Discussion by sunyeongchoi** [react-hook-form/react-hook-form#7749] [Q&A] [answered]
Controlled inputs don't work with [reset](https://react-hook-form.com/api/useform/reset) and [resetField](https://react-hook-form.com/api/useform/resetfield) method

is this an error? Or is the usage wrong?

codesandbox : https://codesandbox.io/s/react-hook-form-controller-reset-bilgq?file=/src/LoginForm.jsx

**Comment by Moshyfawn** [comment, marked as answer]
You need to provide the `<Controller />` props to your input first, before React Hook Form can do anything with your input
```js
render={({ field }) => (
  <input value={field.value} onChange={field.onChange} />
)}
```
Please, refer to [the `<Controller />` usage docs](https://react-hook-form.com/api/usecontroller/controller) to learn more

**Reply by sunyeongchoi** [reply]
Thanks this has been resolved.
thank you!!
